control-plane/src/health.rs

```diff
@@ -33,7 +34,7 @@ async fn run_ecs_health_check_service(
 
         let combined_log = CombinedHealthCheckLog {
             control_plane: draining_log.clone(),
-            data_plane: draining_log,
+            data_plane: HealthCheckVersion::V1(draining_log),
```

Comment:
This isn't really doing anything aside from satisfying constraints. This and the `skip_deep_healthcheck` clause below will be changed in #229 so somewhat irrelevant.